test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
I was calling `exports.execute` and it simply wasn't executing, no errors were thrown, putting `console.log` throughout the `exports.execute` function logged all the way through....  It was just failing silently somewhere....

:cry: